src/Components/DialPad.js

```diff
@@ -1,41 +1,47 @@
 import React from 'react';
 import '../index.css';
+import PropTypes from 'prop-types';
 import SingleButton from './SingleButton';
 
-function DialPad() {
+// eslint-disable-next-line react/prop-types
+function DialPad({ clickHandler }) {
   return (
     <div>
```

Comment:
**[required]**
Let us fragment `<></>` instead of adding an extra `div`. clearly the `div` here is only there to satisfy the react convention of rendering all element within one element. We can efficiently handle this with fragment to avoid creating extra DOM.